go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/sercand/kuberesolver v2.1.0+incompatible
 	github.com/sirupsen/logrus v1.4.2
 	github.com/stretchr/testify v1.4.0
-	github.com/uber/jaeger-client-go v2.22.1+incompatible
+	github.com/uber/jaeger-client-go v2.28.0+incompatible
```

Comment:
Is this necessary to do ipv4-only?  If yes, please explain; if no, please move to a different PR.

Reply:
no is not necessary, it looks like go mod replace it